africanlii/templates/peachjam/header.html

```diff
@@ -2,45 +2,23 @@
-          <path class="st0" d="M7.9 15.7c.2 0 .2.2.2.2s-.1.3-.2.3-.1 0-.1-.1v-.3s0-.1.1-.1"></path>
-          <path d="M10.9 26.7c-2.1-.9-9.3-4.5-9.3-12.9v-.9c.5-.2 1.7-.5 2.2-.7.7-.2 1.4-.1 2.1.2 1.4.6 2.9.9 4.4 1.1.1 0 0 .4 0 .4h-.2c-.3 0-.5 0-.8-.1-.4 0-.7.1-1.1.1l-.5.1c-.3.1-.7.2-1.1.2l-.4-.1h-.3c-.1 0-.5.3-.6.4s-.2 0-.2.1v4.7H10c.1 0 .3.4.3.4v2.1l-.1.2c0 .1-.1.2-.2.3-.1.1-.2.2-.4.2-1-.3-1.8-.8-2.6-1.5 0 0 .3-.3.3-.4s-.3-.3-.4-.3l-.2-.1-.5-.3c-.1-.1-.6-.5-.8-.5s-.2.2-.2.2v.5s.1.3.1.5.1.4.2.6c0 0 .2.4.3.4s.5-.3.5-.3.6.7.7.8l.6.6c.2.2.7.5 1 .8l.4.2.6.4c.1.1.2.1.3.1.3.2.6.3.9.4h.2c.3-.1.7-.2.9-.4.1 0 .2-.1.3-.1s.4-.2.6-.4l.4-.3c.4-.3.7-.5 1-.8l.6-.6c.2-.2.4-.4.5-.4s.5.3.5.3l.3-.4c.1-.2.1-.4.2-.6s.1-.5.1-.5v-.6c0-.1-.1-.2-.2-.2s-.7.4-.8.5l-.4.3-.2.1s-.4.2-.4.3.3.4.3.4c-.7.7-1.6 1.2-2.6 1.5-.2 0-.3 0-.4-.2-.1-.1-.2-.2-.2-.3l-.1-.2v-2.1l.3-.4h4.8v-4.7c0-.1-.2-.1-.3-.1s-.4-.4-.5-.4h-.4c-.1 0-.3.1-.4.1-.3-.1-.7 0-1-.2-.1-.1-.3-.1-.5-.1-.3-.1-.7-.1-1-.2h-1v-.4c1.5-.1 3-.5 4.4-1 .6-.3 1.4-.4 2.1-.2.5.1 1.7.5 2.1.7v.9c.3 8.2-7 11.9-9.1 12.8" fill="#2b8cbe"></path>
-          <path class="st0" d="m5.7 18.8-.1.2v-4.3l.1-.1v4.2zm.4-4.4v4l-.1.2v-4.1l.1-.1zm-.2.1v4.1l-.1.2v-4.2l.1-.1zm0 4.4.3.1h1.4l.4-.1.7-.2c.2 0 .4-.1.6-.1.4 0 .8 0 1.1.2-.3-.2-.7-.3-1.1-.2-.2 0-.3 0-.5.1h.5c.3 0 .7 0 1 .2-.3-.2-.7-.2-1-.2-.2 0-.4 0-.6.1l-.7.1-.4.1c-.5.1-.9.1-1.4 0 0 0-.1 0-.3-.1zm.2-.4c.5.1 1 .1 1.5 0l1-.2c.3-.1.5-.1.8-.1.5 0 .9.1 1.3.3-.4-.2-.8-.4-1.3-.3-.3 0-.5 0-.8.1l-1 .2c-.4.2-.9.2-1.5 0zm-.1.2c.1 0 .3.1.4.1.4.1.8 0 1.2-.1l1-.2c.2-.1.5-.1.7-.1.4 0 .8.1 1.2.3-.3-.2-.7-.3-1.1-.3-.2 0-.5 0-.7.1l-1 .2c-.4.1-.8.1-1.2 0-.2.1-.4 0-.5 0zm9.9.3c-.5.2-1.1.2-1.7.1l-.3-.1-.7-.1c-.5-.1-1.1-.1-1.6.2.3-.2.7-.2 1-.2h.5c-.5-.1-1-.1-1.5.2.3-.2.7-.2 1.1-.2.2 0 .4 0 .6.1l.7.2.3.1c.5.1.9.1 1.4 0l.2-.3.2-.1-.2.1m-.1-.5c-.5.2-1 .2-1.5 0l-1-.2c-.7-.2-1.4-.1-2 .2.4-.2.8-.4 1.3-.3.3 0 .5 0 .8.1l1 .2c.4.1.9.1 1.4 0zm.2.2c-.5.2-1.1.2-1.7.1l-1-.2c-.6-.2-1.3-.1-1.8.2.3-.2.7-.3 1.1-.3.2 0 .5 0 .7.1l1 .2c.4.1.8.1 1.2.1.2-.1.4-.1.5-.2z"></path>
-    </svg>
+        <img src="{% static 'images/uct-logo.svg' %}" alt="uct-logo" height="30" />
       </a>
+      <div class="d-flex align-content-center">
+        <a href="https://www.afrobarometer.org/impact/informing-the-public-discourse/data-for-governance-alliance/#:~:text=A%20project%20to%20promote%20data,African%20Governance%20Platform%20(AGP)."
```

Comment:
is this the correct link to use for the CSO AGP logo?

Reply:
Drop the `#:~:` bit, so just `https://www.afrobarometer.org/impact/informing-the-public-discourse/data-for-governance-alliance/`